WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/simplepayments/TakePaymentViewModel.kt

```diff
@@ -34,12 +37,21 @@ class TakePaymentViewModel @Inject constructor(
 ) : ScopedViewModel(savedState) {
     private val navArgs: TakePaymentFragmentArgs by savedState.navArgs()
 
+    val viewStateLiveData = LiveDataDelegate(savedState, ViewState())
+    internal var viewState by viewStateLiveData
```

Comment:
🔧 `internal` is not very useful here, probably private is the fit
```suggestion
    private var viewState by viewStateLiveData
```